wordclock_interfaces/templates/app.html

```diff
@@ -0,0 +1,141 @@
+    <script src='https://unpkg.com/vuetify/dist/vuetify.js'></script>
+    <script src='https://unpkg.com/babel-polyfill/dist/polyfill.min.js'></script>
+    <script src="https://unpkg.com/@radial-color-picker/vue-color-picker/dist/vue-color-picker.min.js"></script>
+    <link rel="stylesheet" href="static/vue-color-picker.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css">
+</head>
+
+<body>
+
+    <div id="app">
+        <v-app id="inspire" dark>
+            <v-content>
+                <v-container fluid fill-height>
+                    <v-layout align-center justify-center>
+                        <v-flex xs12 sm8 md4>
+                            <v-tabs dark color="primary" class="elevation-12">
+                                <v-tab ripple height="100%">
+                                    Control
+                                </v-tab>
+                                <v-tab ripple>
+                                    Colours
+                                </v-tab>
+                                <v-tab-item>
+                                    <v-card height=562>
+                                        <v-card-text>
+                                            <v-select v-if="apiData" v-bind:items="apiData.plugins" item-text="pretty_name" item-value="name" v-model="selectedPlugin" label="Select plugin" v-on:change="selectionChanged" single-line bottom></v-select>
+                                        </v-card-text>
+                                        <v-card-actions>
+                                            <v-container fluid text-xs-center>
+                                                <v-layout row>
+                                                    <v-flex xs6>
+                                                        <v-card tile flat>
+                                                            <v-card-text>
+                                                                <v-btn color="primary" v-on:click="buttonClick('left')">Left</v-btn>
+                                                            </v-card-text>
+                                                        </v-card>
+                                                    </v-flex>
+                                                    <v-flex xs6>
+                                                        <v-card tile flat>
+                                                            <v-card-text>
+                                                                <v-btn color="primary" v-on:click="buttonClick('right')">Right</v-btn>
+                                                            </v-card-text>
+                                                        </v-card>
+                                                    </v-flex>
+                                                </v-layout>
+                                                <v-layout row justify-center>
```

Comment:
Is there any chance to place all three buttons in one single row?

Reply:
Done